src/interface/src/app/treatments/treatment-plan-tabs/treatment-plan-tabs.component.scss

```diff
@@ -6,3 +6,7 @@
     height: 100%;
   }
 }
+
+::ng-deep .base-maps-selector summary {
```

Comment:
We need to hide the Map selector summary from out MapBaseLayerComponent since it is not required in this design.

Reply:
hm I say just update MapBaseLayerComponent markup to match the current designs and just remove details/summary

Reply:
Updated

Reply:
I say just remove it, we can always easily add it later